hicommonwealth/commonwealth
# 7053 quill pasting bug

## Link to Issue
Closes: #7053 

## Description of Changes
- Added a helper function on the paste hook to handle the extra \n 
## "How We Fixed It"

- Created a function `handleTextPaste` in the `useQuillPasteText` hook to handle the extra \n 

## Test Plan
- Go to a thread or create a thread
- Copy some string with new lines / breaks in content
- Paste and ensure theres no extra lines
